terraform/label.tf

```diff
@@ -0,0 +1,11 @@
+module "label" {
+  source      = "git::https://github.com/cloudposse/terraform-null-label.git?ref=master"
```

Comment:
I'd like us to vendor this by pinning to a given release/commit for this dependency, in order to avoid upstream breakages.

Reply:
I don't think they've ever released a backwards-incompatible change, but you're probably right to be safer here. Will do.

Reply:
Done